.github/workflows/main.yaml

```diff
@@ -89,6 +89,9 @@ jobs:
         go-version: '1.17'
       id: go
 
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
```

Comment:
we checkout at line no: 84 as well, its about ordering? 🤔

Reply:
Wow, for some reason when I did this I did not see that on line 84 at all! I'll close this PR.

Though that still doesn't explain why the other jobs failed to see the changes in the code when running linting. I guess that will require further investigation.